content/en/docs/concepts/instrumentation/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Instrumentation
+description: >-
+  How OpenTelemetry facilitates automatic and manual instrumentation of
+  applications.
+aliases: [/docs/concepts/instrumenting/]
+weight: 15
```

Comment:
Use a weight so that this new section appears after "Componets" or, like it was before, after semconv?

Reply:
My assumption was that "Instrumentation" is something people look for before going into components and/or SemConv, but that's just a feeling, wdyt? @chalin @cartermp

Reply:
I actually like this where it is, since it's such a fundamental grouping of topics.

Reply:
> I actually like this where it is, since it's such a fundamental grouping of topics.

so you are against the PR overall or against changing the oder of things, meaning we keep Signals, Components, SemConv & then Instrumentation?

Reply:
Sorry, what I meant was I like this current change -- Instrumentation feels like it belongs up high like this.

Reply:
so we keep it that way :-)